Client/AccountSyncHandler.swift

```diff
@@ -4,6 +4,7 @@
 
 import Foundation
 import Storage
+import Common
```

Comment:
Is common replacing shared in the future or they will mostly live together to serve a different purpose?

Reply:
It's possible eventually there will be no more use for shared but it's likely a very long time down the road.